tests/tasks/Cloudwatch/cw-metric.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name:  cloudwatch
+  namespace: tekton-pipelines
+spec:
+  description: "cw task to push metrics relying on generate task."
+  params:
+    - name: region
+      default: "us-west-2"
+    - name: desired-nodes
+    - name: load-result
+  steps:
+  - name: cw-emit
+    image: alpine/k8s:1.22.6
```

Comment:
We don't depend on any k8s stuffs in this task. Why don't we use image `amaon/aws-cli`?

Reply:
yes correct will change it